### ISSUE TITLE
Filter out Users who cannot login

### ISSUE THREAD

**Issue by igalic** [elementary/switchboard-plug-useraccounts#136]
## Prerequisites
- [x] I have searched open and closed issues for duplicates.

## Describe the bug
On the login screen, as well as in the switchboard plug for user accounts, we get a listing of users who cannot login.

If applicable, add screenshots or screen recordings to help explain your problem.

![Example showing Libvirt Qemu User in the Switchboard](https://user-images.githubusercontent.com/497249/78401252-a2aff080-75f8-11ea-88a1-c5323811107f.png)

### To Reproduce
install `virt-manager`
reboot: Login shows Libvirt Qemu user

note that this user is actually disabled!

here's passwd:

```
libvirt-qemu:x:64055:129:Libvirt Qemu,,,:/var/lib/libvirt:/usr/sbin/nologin
libvirt-dnsmasq:x:123:132:Libvirt Dnsmasq,,,:/var/lib/libvirt/dnsmasq:/usr/sbin/nologin
```

From my understanding the reason, the reason why this user is shown is because it's not a system user (uid >= 1000).

### Expected behavior
I'd expect users with an invalid shell to not show up in the listing.
Disabling a user, i suppose, doesn't change the shell, and instead uses `passwd -l` to lock the user.

## Platform Information
![image](https://user-images.githubusercontent.com/497249/78402134-6087ae80-75fa-11ea-9392-a9a1e89084aa.png)

 - OS: elementary OS
 - OS Version: Hera
 - Hardware info: Intel i7…

- [ ] I'm using the latest version from git that I've manually compiled
- [x] I'm using the latest released stable version

**Comment by danrabbit** [comment]
This is an issue in an upstream component regarding `nologin` users. It is fixed in newer versions of that component, but it doesn't seem like there will be a backport to the Ubuntu 18.04 repos.

Closing as "Out of scope" since the issue is with an upstream component

**Comment by igalic** [comment]
aye
so the issue should be solved as soon as elementary updates to 20.04…?

**Comment by danrabbit** [comment]
Yup exactly
